src/ssl.c

```diff
@@ -48858,7 +48897,11 @@ int wolfSSL_SSL_do_handshake(WOLFSSL *s)
 #endif
 }
 
+#if defined(WOLFSSL_QT)
```

Comment:
Use OPENSSL_VERSION_NUMBER instead of WOLFSSL_QT

Reply:
Guard by OPNESSL_VERSION_NUMBER